src/yunohost/app.py

```diff
@@ -3636,7 +3636,11 @@ def _patch_legacy_helpers(app_folder):
         if not os.path.isfile(filename):
             continue
 
-        content = read_file(filename)
+        try:
+            content = read_file(filename)
+        except Exception:
```

Comment:
I suggest this change (should be better than general Exception)
```suggestion
        except MoulinetteError:
```
Currently it means: "file_not_exist", "cannot_open_file" and "unknown_error_reading_file"
See https://github.com/YunoHost/moulinette/blob/dev/moulinette/utils/filesystem.py#L34